### ISSUE TITLE
Example pull request for a test case

### ISSUE THREAD

**Issue by jmlane** [jmlane/scribe-abacus#5]
Create an example of using Github features to accept contributions to the project.

**Steps:**
- [x] Create this [Issue].
- [x] [Edit] the [test suite] and add a new test case.
- [x] Create a [Pull Request] (PR) to submit the changes to the project authors.
- [x] Demonstrate PR threaded discussion/commenting and [reviewing changes][diff] features.
- [x] Accept PR and outline how accepted changes are integrated.
- [x] *Finally*, [explain how the test cases are formatted][test case], since that's the whole point!

[Issue]: https://help.github.com/articles/about-issues/
[Edit]: https://help.github.com/articles/editing-files-in-another-user-s-repository/
[test suite]: https://github.com/jmlane/scribe-abacus/blob/master/tests/Tests.elm
[Pull Request]: https://help.github.com/articles/about-pull-requests/
[diff]: https://help.github.com/articles/reviewing-proposed-changes-in-a-pull-request/
[test case]: https://github.com/jmlane/scribe-abacus/pull/7/files#r101706075

**Comment by jmlane** [comment]
**TLDR version**
1. Open the [tests/Tests.elm](https://github.com/jmlane/scribe-abacus/blob/master/tests/Tests.elm) file
2. Click edit (pencil icon) on the upper right
3. Check df39590 for edit example (called a commit)
4. Check Pull Request #7 created from the edit form submission
5. Read the PR for context and more details
6. Read [the test commit comment](https://github.com/jmlane/scribe-abacus/pull/7/files#r101706075) that explains how the test is formatted and how to create a new test.
7. ???
8. Profit

In order to make an edit, select a file from the [Code](https://github.com/jmlane/scribe-abacus) tab at the top of the page. In our context, we want to edit [tests/Tests.elm](https://github.com/jmlane/scribe-abacus/blob/master/tests/Tests.elm) which I'll link here for convenience.

Once you have found the file you wish to edit, you can click the "edit this file" (pencil icon) button on the top right corner of the file panel. This will bring you to a file editing form which allows you to make changes directly to the file without downloading anything or setting up Git on your computer. It is useful for making minor edits from the browser.

At the bottom of this form, there are fields for a commit message, commit details, and an option for what you want to do with edit. We are interested in making a *Pull Request* so you will need to select that option and create a new branch. Giving it a contextual name helps everyone keep track of distinct sets of changes but it isn't critical to do so.

An example of a good commit message can be found on the example commit I made: df39590

**Comment by jmlane** [comment]
Given the example code was merged, this issue could be closed. I will leave it open for questions and discussion.
